Bookish.Web/Views/Home/Copies.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@model CopiesViewModel
+@{
+    ViewData["Title"] = Model.Title;
+    ViewData["Author"] = Model.Author;
+}
+
+<link rel="stylesheet" href="@Url.Content("~/css/home/copies.css")" asp-append-version="true">
```

Comment:
Maybe I'm being blind but I can't see the copies.css file here in these changes? 🙂

Reply:
I forgot to stage the file, although .book-table doesn't actually define any properties since it gets all the styling from .table in bootstrap.

Reply:
Ah I see! Sorry, I misunderstood earlier. I thought you were trying to add your own styling to the table which was why I was suggesting the new css class. If you're not then you can just use the bootstrap table class. Sorry about that, my mistake for trying to review quickly last night and not reading things properly! 🙁

Reply:
No worries!